parm/config/config.ocnpost

```diff
@@ -0,0 +1,14 @@
+#!/bin/ksh -x
+
+########## config.ocnpost ##########
+
+echo "BEGIN: config.ocnpost"
+
+# Convert nemsio files to grib files using post job
+#-------------------------------------------
+
+# No. of concurrent post jobs [0 implies sequential]
+export NPOSTGRP=1
+export ncks="/apps/intel-2020/nco-4.8.1/bin/ncks"
```

Comment:
This is hardwired for Orion.
Should be handled by a modulefile and not in `config.ocnpost`

Reply:
There's a nicks for each platform in for example: https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/workflow/platforms/orion.yaml#L42 or https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/workflow/platforms/hera.yaml#L41

Reply:
The nco module in hpc-stack adds the bin directory to the path, so we should be able to just set this to "ncks". Or use $(which ncks) if we really want the full path.

Reply:
Rahul is right, this should be covered by the module, not a path we set for each platform.

Reply:
So remove the ncks line completely from config.ocnpost?

Reply:
Set it to ncks. Otherwise you'll have to replace the $ncks wherever it is used in ocean post.

Reply:
`export ncks="ncks"`?

Reply:
IMO, this line should disappear from config files altogether.

Wherever it is used, it should be resolved as `$(which ncks)` and then used as appropriate.
If this is used in ocnpost, we have the chance to clean than up right now.  

I suggest removing this line completely.

Reply:
Will remove line.